### PR TITLE
chore(tags): add macosarmjava tag

### PR DESCRIPTION
**Pull request by Lutitious**
Add a tag to link to downloads for MacOS arm Java for azul

### CODE REVIEW

File: tags/macosarmjava.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Install Arm Java on MacOS
+color: yellow
+aliases: ['armmacosjava']
+---
+
+On MacOS Arm, also known as Apple Silicon (M1, M2, etc.), you will need to install The Arm Version of Java. We recommend using Azul, the links for which can be found below:
+
+- [Java 8](https://aka.ms/vs/17/release/vc_redist.x64.exe) for versions ≤ 1.16
```
Comment (TayouVR) [inline tags/macosarmjava.md:9]: 👀

Comment (Lutitious) [reply tags/macosarmjava.md]: Oops

File: tags/macosarmjava.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Install Arm Java on MacOS
+color: yellow
+aliases: ['armmacosjava']
+---
+
+On MacOS Arm, also known as Apple Silicon (M1, M2, etc.), you will need to install The Arm Version of Java. We recommend using Azul, the links for which can be found below:
```
Comment (ryanccn) [inline tags/macosarmjava.md:7]:
```suggestion
On macOS arm64, also known as Apple Silicon (M1, M2, etc.), you will need to install the arm64 version of Java. We recommend using builds from Azul, the links for which can be found below:
```

File: tags/macosarmjava.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- [Java 8](https://www.azul.com/downloads/?version=java-8-lts&os=macos&architecture=arm-64-bit&package=jdk#zulu) or versions ≤ 1.16
+- [Java 17](https://www.azul.com/downloads/?version=java-17-lts&os=macos&architecture=arm-64-bit&package=jdk#zulu) for versions ≥ 1.17
+
+It is recommended to install both Java 8 and Java 17
```
Comment (ryanccn) [inline tags/macosarmjava.md:12]:
```suggestion
It is recommended to install both Java 8 and Java 17.
```

File: tags/macosarmjava.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: Install Arm Java on MacOS
```
Comment (ryanccn) [inline tags/macosarmjava.md:2]:
```suggestion
title: Install arm64 Java on macOS
```

**Review by ryanccn — CHANGES_REQUESTED** [review]

**Review by ryanccn — APPROVED** [review]

**Comment by Scrumplex** [comment]
Why wasn't this part of the Java tag?

**Comment by Lutitious** [comment]
> Why wasn't this part of the Java tag?

I didn't think of that tbh

**Pull request merged**